index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="Header">
+   <header>
+    <nav class="nav-bar">
+        <a href="#">Brenda Lutiku</a>
+        <img src="./images/Enabled.png" alt="" class="menu">
+    </nav>
```

Comment:
- [ ] I think it is better to wrap only the hamburger menu inside ``nav``
```suggestion
        <a href="#">Brenda Lutiku</a>
     <nav>
        <img src="./images/Enabled.png" alt="" class="menu">
    </nav>
```